docs/Contributing/Feature Requests/Feature_requests.md

```diff
@@ -0,0 +1,6 @@
+## Feature requests
+
+Feature requests are welcome. But take a moment to find out whether your idea
+fits with the [scope and aims](https://atos.readthedocs.io/en/latest/) of the project. It's up to *you* to make a strong
```

Comment:
The linked text does not exactly and clearly describe the scope and aims of ATOS.
Might be a good idea to very clearly state what they are, maybe under its own heading. 
But can be done in another PR.

Reply:
We can add this as a new section. Do you have a nice crispy definition of the scope and aim that is used elsewhere?

Reply:
Nope, not right now, we have to come up with some clear definitions of long-term goals for the project